pkgs/top-level/all-packages.nix

```diff
@@ -7885,10 +7885,10 @@ with pkgs;
 
   volctl = callPackage ../tools/audio/volctl { };
 
-  volk = if (stdenv.isDarwin && stdenv.isAarch64) then
-    (callPackage ../development/libraries/volk/2.5.0.nix { })
+  libvolk = if (stdenv.isDarwin && stdenv.isAarch64) then
```

Comment:
I had to resolve a naming conflict. Renaming this to libvolk seems appropriate: https://www.libvolk.org/